fix_appdata_file.patch

```diff
@@ -0,0 +1,47 @@
+diff -ruN a/Clients/ParaView/org.paraview.ParaView.appdata.xml b/Clients/ParaView/org.paraview.ParaView.appdata.xml
+--- a/Clients/ParaView/org.paraview.ParaView.appdata.xml	2021-01-24 01:01:44.000000000 +0700
++++ b/Clients/ParaView/org.paraview.ParaView.appdata.xml	2021-03-27 20:30:00.876773584 +0700
+@@ -3,9 +3,12 @@
+ <component type="desktop-application">
+  <id>org.paraview.paraview</id>
+  <metadata_license>CC-BY-4.0</metadata_license>
+- <project_license>BSD</project_license>
++ <project_license>BSD-3-Clause</project_license>
+  <name>ParaView</name>
+  <summary>Data analysis and visualization</summary>
++ <content_rating type="oars-1.1"/>
++ <releases>
++     <release version="5.9.0" date="2021-01-28">
```

Comment:
This cause the file to be invalid xml. `<release>` and `<releases>` are not closed. Hence the build failure.